app/models/users/permission_checks.rb

```diff
@@ -56,6 +56,15 @@ def allowed(action, project)
     def allowed_members(action, project)
       Authorization.users(action, project).where.not(members: { id: nil })
     end
+
+    def allowed_on_work_package(action, work_package)
```

Comment:
Can we rename this to `allowed_members_on_work_package`? It returns a list of members, and that should be representend in the name

Reply:
I thought of it but couldn't decide on which of the two method names to pick. Thanks